great_expectations_provider/operators/great_expectations.py

```diff
@@ -21,7 +21,9 @@
 from datetime import datetime
 from typing import TYPE_CHECKING, Any, Callable, Dict, List, Optional, Union
 
+import airflow.providers.snowflake  # noqa
```

Comment:
This makes snowflake provider a hard dependency for installing great-expectations.

Do we really want that?

Reply:
It is not used anywhere as well, why do we need this import line?